consensus/safety-rules/src/process.rs

```diff
@@ -18,7 +19,8 @@ pub struct Process {
 
 impl Process {
     pub fn new(mut config: NodeConfig) -> Self {
-        let (author, storage) = safety_rules_manager::extract_service_inputs(&mut config);
+        let (author, storage, execution_pubkey) =
+            safety_rules_manager::extract_service_inputs(&mut config);
```

Comment:
we should not be receiving an execution pub key, we should retrieve it from storage within safety rules -- what we should potentially extract is whether or not we should expect signed vote proposals